manifests/init.pp

```diff
@@ -46,6 +47,7 @@
   Stdlib::Absolutepath $configdir,
   Stdlib::Absolutepath $rubypath,
   Boolean $manage_bin_symlinks = false,
+  Boolean $manage_facts_refresh = true,
```

Comment:
instead of a new boolean here can we add `none` to the enum of `$facts_refresh_type`? And make the enum double quotes

Reply:
@ripienaar This is done.